### ISSUE TITLE
Structural Genomics - genera supported by the 2 centers needs updating

### ISSUE THREAD

**Issue by rwill1** [PATRIC3/patric3_website#370]
In the 1st contract, each of the two centers had a non-intersecting set of organisms (spread across the BRCs) that they supported.   See the list at https://www.ssgcid.org/community-requests/  They requested that for proteins in those organisms that didn't currently have a structure, that we support the ability for a PATRIC user to easily submit a Community Target Request for structure determination by filling in info that PATRIC has (like IDs and sequence data) so that the user didn't have to.  Since that time, it seems that the 1) organisms have become non-exclusive between the 2 centers (e.g. SSGCID is working on Escherichia according to their home page AND their Target Status Page, even though it is listed as a CSGID organism on the above link).  In addition, SSGCID is working on structures for organisms that are not on their home page and not in the segregated organism list (e.g.Pseudomonas, Acinetobacter - per emails from Isabelle and per their Target Status Page).   Therefore, if we are going to continue to facilitate PATRIC users submitting community requests to the centers, we need an updated list for how to distributed the requests and what genera to support the requests for.

In addition, it's not clear if our incorrect info on what genera each center supports prevents us from displaying data even if we've picked it up.   For example
Pseudomonas https://apps.sbri.org/SSGCIDTargetStatus/Target/PsaeA.00023.a
has a pointer to PATRIC and says most recent activity was in late 2014 and I THINK we have structures submitted to PDB later than that with PATRIC pointers to SSGCID.   But when you visit that feature at PATRIC https://www.patricbrc.org/portal/portal/patric/Feature?cId=PATRIC.208964.12.NC_002516.CDS.4512549.4513076.fwd&cType=feature the structure field says "Not supported by SSGCID/CSGID"

**Comment by hyoo** [comment]
I don't think this is a bug, but a requirement change. We need to update database/solr and change logic behind it. I would like to do this in Nov release. Please comment if you think this is urgent.

**Comment by rwill1** [comment]
Maulik now is responsible for the relationship with SSGCID.   Rick was responsible for restarting the relationship with CSGID if he thought it was worthwhile.  So I'd say that at this point it's up to Maulik to make the call on how urgent this is.    I think the impact is 1) we're probably not linking to SSGCID for structures that we do have from them (if we pick up new structures this release)  2) we're not enabling sending community requests for some genera (I don't know how many people actually ever did this)  3) there are maybe a few community requests going to one center that should be going to the other.  My guess is that 1 is really the only important issue and that it can wait.

**Comment by mshukla1** [comment]
I have send a mail to Isabelle Phan regarding the list of genera we should be using for linking to SSGCID structure request form. Waiting for response. 

-Maulik
